### ISSUE TITLE
Pose transfer

### ISSUE THREAD

**Issue by Mathilda88** [AliaksandrSiarohin/first-order-model#352]
Hi Aliaksandr,

Thanks for sharing the great work, I was wondering what happened if we just used a single image as the driving sample rather than the video. What does the output look like in this case?

Thanks,

**Comment by AliaksandrSiarohin** [comment]
If the mode is relative is will be same as source. I the mode is absolute it will be source in the driving pose.

**Comment by Mathilda88** [comment]
Thank you for the response. Actually, I tried the model in the absolute mode but the results are quite gibberish with a halo of similarity to a human face (especially when the poses are not much similar). May I have your thought about that?

**Comment by AliaksandrSiarohin** [comment]
It also transfer shape of the underlying object, you check in colab that it is trying somehow to change the shape of putin haircut to match trump haircut.

**Comment by Mathilda88** [comment]
Yes, but it seems to be in relative mode (I mean the model used for training).

**Comment by AliaksandrSiarohin** [comment]
Don't get what you mean in training there is no difference.
